contracts/MerkleProof.sol

```diff
@@ -68,4 +68,29 @@ library MerkleProof {
 
     return result[0] == 0x01;
   }
+
+  /**
+   * @dev Returns true if a `leaf` can be proved to be a part of a Merkle tree
+   * defined by `root`. For this, a `proof` must be provided, containing
+   * sibling hashes on the branch from the leaf to the root of the tree. Each
+   * pair of leaves and each pair of pre-images are assumed to be sorted.
+   */
+  function verify(bytes32[] memory proof, bytes32 root, bytes32 leaf) internal pure returns (bool) {
```

Comment:
Which code do you refer to?

Reply:
it's copied from `@openzeppelin/contracts`
 https://www.npmjs.com/package/@openzeppelin/contracts/v/3.4.2?activeTab=code